pages/common/fd.md

```diff
@@ -10,11 +10,11 @@
 
 - Find files that begin with `foo`:
 
-`fd "^foo"`
+`fd "{{^foo}}"`
```

Comment:
```suggestion
`fd "^foo"`
```
In this case it is not a placeholder because of the command description.

Reply:
I think this should be a user-provided value and in #15037 @einverne suggested that I should change it like this.